site/docs/use-cases/image-processing/_sections/_run_model/index.mdx

```diff
@@ -1,10 +1,12 @@
 import CodeExampleCPP from './_code_example_cpp.mdx';
 import CodeExamplePython from './_code_example_python.mdx';
 
-## Run Model Using OpenVINO™ GenAI
+## Run Model Using OpenVINO GenAI
 
-OpenVINO GenAI introduces `Text2ImagePipeline` for inference of text-to-image models such as Stable Diffusion 1.5, 2.1, XL, LCM, Flux, and more.
-See all supported [image generation models](/docs/supported-models/#image-generation-models).
+OpenVINO GenAI introduces the `VLMPipeline` pipeline for inference of multimodal text-generation Vision Language Models (VLMs).
```

Comment:
In a spirit of adding links everywhere, maybe class introductions mentioning its name for the first time could point to its C++ header implying that the docs for the class can be found there. On the other hand adding too much links can be overwhelming.

Do you have thoughts on that?

Reply:
I agree that it will be helpful to navigate user to pipeline class API description. And in my opinion one more link here won't be so overwhelming.
I would suggest to add link to https://docs.openvino.ai/2025/api/genai_api/_autosummary/openvino_genai.VLMPipeline.html instead of raw C++ header file.
I understand that C++ header file is more actual than autogenerated API reference page, but it might be more confusing for the user (e.g. python user) to deal with raw C++ code and unformatted comments and annotations.